service/policy/kasregistry/key_access_server_registry.proto

```diff
@@ -19,11 +22,19 @@ message ListKeyAccessServersResponse {
   repeated KeyAccessServer key_access_servers = 1;
 }
 
+// TODO: optional validation below should be through a custom validator, which is too bleeding edge at present without full plugin support
+
 message CreateKeyAccessServerRequest {
   // Required
-  string uri = 1 [(buf.validate.field).required = true];
+  string uri = 1 [(buf.validate.field).cel = {
+      id: "uri_format",
+      message: "URI must be a valid URL (e.g., 'https://demo.com/') followed by additional segments. Each segment must start and end with an alphanumeric character, can contain hyphens, alphanumeric characters, and slashes.",
+      expression: "this.matches('^https?://[a-zA-Z0-9]([a-zA-Z0-9\\\\-]{0,61}[a-zA-Z0-9])?(\\\\.[a-zA-Z0-9]([a-zA-Z0-9\\\\-]{0,61}[a-zA-Z0-9])?)*(:[0-9]+)?(/.*)?$')"
```

Comment:
@dmihalcik-virtru Hey Dave, I noticed we weren't doing validation on this field previously and some of our test pipelines were creating KASes with URIs that didn't use TLS. Does updating the URI format for the registered KASes throughout to `http` make sense to you?

Reply:
We will allow TLS as optional per existing standards in the OpenTDF Platform